lib/dataContract/validateDataContractFactory.js

```diff
@@ -4,7 +4,7 @@ const ValidationResult = require('../validation/ValidationResult');
 const DataContract = require('./DataContract');
 const generateDataContractId = require('./generateDataContractId');
 
-const baseDocumentSchema = require('../../schema/base/document');
+const baseDocumentSchema = require('../../schema/document/documentBase');
```

Comment:
Why not just `/base` as you've done with `documentTransition` below?

Reply:
Resolved https://github.com/dashevo/js-dpp/pull/135#discussion_r398018787